pkgs/top-level/all-packages.nix

```diff
@@ -19601,7 +19601,11 @@ in
 
     rtl8821cu = callPackage ../os-specific/linux/rtl8821cu { };
 
-    rtlwifi_new = callPackage ../os-specific/linux/rtlwifi_new { };
+    rtw88 = if lib.versionAtLeast kernel.version "4.14"
+    then callPackage ../os-specific/linux/rtw88 { }
+    else (callPackage ../os-specific/linux/rtw88 { }).overrideAttrs (_: { meta.broken = true; });
```

Comment:
Why are you not setting meta broken in the package itself? An example https://github.com/NixOS/nixpkgs/blob/8cb1afae1b57daeeb248ce54534398ccb9fd4e2d/pkgs/os-specific/linux/ena/default.nix#L44

Reply:
Thats much cleaner, I don't know why I didn't think of that.
Thanks for taking the time to review this btw!